### PR TITLE
Adding tests of FullChem

### PR DESCRIPTION
**Pull request by jialinl6**

### CODE REVIEW

File: test/geoschem_test.jl [hunk 1]
```diff
@@ -0,0 +1,86 @@
+using GasChem
+using Test
+using OrdinaryDiffEq, ModelingToolkit
```
Comment (ctessum) [inline test/geoschem_test.jl:3]: We can use `DifferentialEquations` instead of `OrdinaryDiffEq` here because it is already installed.

**Review by ctessum — CHANGES_REQUESTED** [review]
Thank you, looking good!

File: Project.toml [hunk 2]
```diff
@@ -6,8 +6,11 @@ version = "0.2.0"
 [deps]
 Catalyst = "479239e8-5488-4da2-87a7-35f2df7eef83"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
 EarthSciMLBase = "e53f1632-a13c-4728-9402-0c66d48804b0"
 ModelingToolkit = "961ee093-0014-501f-94e3-6117800e7a78"
+OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
+SafeTestsets = "1bc83da4-3b8d-516f-aca4-4fe02f6d838f"
```
[the remark this answers is not in the document]
Comment (jialinl6) [reply Project.toml:13]: I see!! Thank you for pointing it out! Finally understand this file.

File: test/geoschem_test.jl (hunk 1 above)
Comment (jialinl6) [reply test/geoschem_test.jl:3]: Right!

**Review by ctessum — APPROVED** [review]
Thanks!

**Pull request merged**